### ISSUE TITLE
Coop Death Fizzler: one that will affect P-body only, and other for Atlas only.

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#3266]
The P-Body´s death fizzler should probably look yellow/orange and blue for Atlas´s death fizzler.

**Comment by The-Queen55142** [comment]
This is currently being developed within a user-created package bY Flash4433 in Enderzombie's retro package. I'm sure once they finish, if you ask them and whoever Pusher71 is, you could get it into default BEE2

**Comment by Flash4433** [comment]
Yes, they're called Co-Op gates. Although they work a little differently (only atlas can pass through blue, only P-body can pass through orange, the other player simply dies), they can be easily changed to serve this purpose. It even comes with custom models and materials. 

I tried to make it work with BEE2, however, whenever I try to compile a map with it P2 just freezes. Not sure what's the problem, perhaps something on the vbsp_config.cfg.

**Comment by ZombiePwner1** [comment]
@Flash4433 I figured I knew my info was incorrect thought it was Atlas that dies when he enters orange but it's blue instead.
